lisa/operating_system.py

```diff
@@ -372,7 +373,16 @@ def _package_exists(self, package: str, signed: bool = True) -> bool:
 
     def _get_os_version(self) -> OsVersion:
         os_version = OsVersion("")
-        cmd_result = self._node.execute(cmd="lsb_release -as", no_error_log=True)
+        # Output of 'lsb_release -as' doesn't match the pattern.
+        # No LSB modules are available.
+        # Debian
+        # Debian GNU/Linux 10 (buster)
+        # 10
+        # buster
+        # Change it into 'lsb_release -ar'.
+        cmd_result = self._node.execute(
```

Comment:
This is what I see from the man page - 

-v, --version
Show the version of the LSB against which your current installation is compliant. The version is expressed as a colon separated list of LSB module descriptions.
-i, --id
Display the distributor's ID.
-d, --description
Display a description of the currently installed distribution.
-r, --release
Display the release number of the currently installed distribution.
-c, --codename
Display the code name of the currently installed distribution.
-a, --all
Display all of the above information.
-s, --short
Use the short output format for any information displayed. This format omits the leading header(s).

Are we able to avoid the issue with the first line in the output with option "-ar"?

Reply:
yeah, I will use -a only